### DISCUSSION TITLE
[Feedback Wanted] Feed bug from 11/4/2024 - Feed showing user activities from users not followed.

### DISCUSSION THREAD

**Discussion by LiteBrite82** [community/community#143676] [Other Feature Feedback, Questions, & Ideas]
Hi Community!

Thank you all for your posts and comments on the recent Feed bug. The bug was related to starred repositories, which is why we see some people saying the fix is to uncheck "Include events from starred repositories". 

The Feed team has pushed an update to address the issue; please leave us feedback in this post so we can monitor its effectiveness.

**Comment by sandros94** [comment]
Working as intended now

**Comment by ghost** [comment]
I'm still receiving notifications about every merged MR in `NixOS/nixpkgs` (which i starred) (last one 1 minute ago).  
But the `xy starred id-software/DOOM` is gone.

**Comment by bkbartk** [comment]
Disabling “repository activity” fixes the issue for me. 
this setting wasn’t there earlier.

**Reply by AlessandroMiola** [reply]
Same for me, thanks!

**Reply by haydenmarshall65** [reply]
This option did fix a lot for me but it has prevented me from seeing version updates from my starred repositories so I only see updates from users I follow instead

**Reply by ribezzz** [reply]
Unfortunaly, disabling “repository activity” doesn't fix the issue for me

**Reply by zqpvr** [reply]
I still have this issue.

**Reply by KieranP** [reply]
Until Github fixes this critically broken part of their app, for those who are growing weary of not being able to keep up with recent releases for starred repositories, I have created a service that does just that.

It is available here: **https://kieranp.github.io/Github-Releases-Feed/**

The service is open sourced under GPLv3 here: https://github.com/KieranP/Github-Releases-Feed

Hope this is useful to others. If you find it useful,

**Comment by eitsupi** [comment]
It seems that the spam postings disappeared for me when the repository activity was deactivated.

**Comment by Bernankez** [comment]
Now the "who followed the reponsitory" has disappeared, but my feed is flooded with MR activities, which wasn't the case before.
![image](https://github.com/user-attachments/assets/ed21effe-54d0-41a0-a7b4-ece3be6a9202)

**Reply by thomasfire** [reply]
Same, still annoying, but I do not want to disable the activities, because I want to see if somebody releases something

**Reply by striezel** [reply]
Exactly. Merge requests are now flooding everything. This is not good, because it buries important activities like releases way down in the feed.

I guess the proper way should be to not show merge request activity in the feed, just as it has been before, and provide a way for GitHub users to opt-in for merge requests. That way most users can keep the usual feed and those who want to see the merge requests can enable them explicitly for their feed.

**Reply by KieranP** [reply]
Until Github fixes this critically broken part of their app, for those who are growing weary of not being able to keep up with recent releases for starred repositories, I have created a service that does just that.

It is available here: **https://kieranp.github.io/Github-Releases-Feed/**

The service is open sourced under GPLv3 here: https://github.com/KieranP/Github-Releases-Feed

Hope this is useful to others. If you find it useful,

**Comment by LuciusChen** [comment]
I think that GitHub, as an infrastructure of the Internet, any major changes should be notified to users in advance, providing necessary switches for users to choose their actions.

**Comment by ADD-SP** [comment]
Still receiving notification of merged PR from stared repository.

**Reply by ADD-SP** [reply]
And also still receiving notification from non-followed users.

**Comment by Nahuel92** [comment]
This keeps happening for me. I still see merged PR from other people (which I don't care about) on starred repos. Please fix it and bring back my clean feed page

**Reply by Nahuel92** [reply]
It seems to be fixed now. Thanks!

**Reply by KieranP** [reply]
Until Github fixes this critically broken part of their app, for those who are growing weary of not being able to keep up with recent releases for starred repositories, I have created a service that does just that.

It is available here: **https://kieranp.github.io/Github-Releases-Feed/**

The service is open sourced under GPLv3 here: https://github.com/KieranP/Github-Releases-Feed

Hope this is useful to others. If you find it useful,

**Comment by om3ga6400** [comment]
great now I can either get spammed or not get notified of new releases, such a good update

**Reply by KieranP** [reply]
Until Github fixes this critically broken part of their app, for those who are growing weary of not being able to keep up with recent releases for starred repositories, I have created a service that does just that.

It is available here: **https://kieranp.github.io/Github-Releases-Feed/**

The service is open sourced under GPLv3 here: https://github.com/KieranP/Github-Releases-Feed

Hope this is useful to others. If you find it useful,

**Comment by lloydrichards** [comment]
Am also experiencing the issue with my feed being flooded with contribution to repos i've starred but from users i'm not following.  As this is a thread for feedback, my expected behaviour would be to see stars/forks from users I've followed and see the releases from repos I've starred and contributions from users I've followed to repos I've starred:

```mermaid
graph LR
  subgraph follow
    User1
    User2
   Team1
  end
  
  subgraph star
    Repo1
  end

   
   User1 --forked-->Repo2
   User2 --starred-->Repo3
   User1 --contributed-->Repo1
  Team1 --created-->Repo4
```
But I shouldn't be seeing contributions from users I've not followed to help reduce the flood

**Comment by luffy-yu** [comment]
IT'S STILL HAPPENING!

**Reply by luffy-yu** [reply]
It seems not showing up now.

**Comment by forest1102** [comment]
Spammed by MR, or no notification, your choice

**Reply by om3ga6400** [reply]
fr

**Comment by imHaruuu** [comment]
![Screenshot 2024-11-07 004729](https://github.com/user-attachments/assets/461adf08-2fc0-4307-a510-e74451e092c5)
but we still have another problem...

**Comment by om3ga6400** [comment]
did they finally fix it??? I'm not getting anymore bs notifcations

**Reply by lloydrichards** [reply]
same 🤞

**Reply by luffy-yu** [reply]
same

**Comment by Challenger1005** [comment]
I need answer to this question

**Reply by om3ga6400** [reply]
its fixed for me

**Comment by hundehausen** [comment]
I still don't see **releases** in my feed. But I can see other types of activities from my starred repositories.

**Comment by prymitive** [comment]
My feed is spammed with a flood of “dependabot contributed to your repository” posts, which I can’t find any way of turning off.

**Comment by AgataKrist** [comment]
Hi

**Reply by om3ga6400** [reply]
hello

**Comment by xyloflake** [comment]
Not fixed for me; it was fine a week ago but now I see nothing except follows which is horrendously weird.

**Comment by ArcaneDisgea** [comment]
This has started happening to me the last few days as well. The only filters I have set are:
- Releases
- Sponsors
- Stars
- Repositories
- Follows
- Recommendations

But I am only seeing a single follow event (user following me) and a single star event (user starring my repo).

I am only following 30 users total since that was questioned earlier, 2 of which being orgs. Checking the [hidden dashboard](https://github.com/dashboard-feed) there are roughly 21 events chronologically since the single follow event that should be showing alongside it.

This happens on different computers entirely, I have tried the whole clear cache thing. The machine I am writing this from, which is affected, is an M1 Max Macbook Pro and Chrome version is `131.0.6778.86 (Official Build) (arm64)` if any of that helps.

Github was a daily check for me because I like seeing what my friends are up to but as long as this bug is around it makes it much harder and much less useful for that.

**Reply by ArcaneDisgea** [reply]
Just updating that it is suddenly working now. Not sure if anything was done on the backend based on this report or just coincidence but thank you nonetheless.

**Reply by ArcaneDisgea** [reply]
Yeah, mines broken again. Seems like it happens almost daily around noon for me and then resolves itself whenever it feels like it.

**Reply by KieranP** [reply]
Until Github fixes this critically broken part of their app, for those who are growing weary of not being able to keep up with recent releases for starred repositories, I have created a service that does just that.

It is available here: **https://kieranp.github.io/Github-Releases-Feed/**

The service is open sourced under GPLv3 here: https://github.com/KieranP/Github-Releases-Feed

Hope this is useful to others. If you find it useful,

**Comment by NicoCaldo** [comment]
still having issue. The only things I see on the feed are Follows and Recommendations

**Reply by KieranP** [reply]
Until Github fixes this critically broken part of their app, for those who are growing weary of not being able to keep up with recent releases for starred repositories, I have created a service that does just that.

It is available here: **https://kieranp.github.io/Github-Releases-Feed/**

The service is open sourced under GPLv3 here: https://github.com/KieranP/Github-Releases-Feed

Hope this is useful to others. If you find it useful, feel free to star the repo :-)

**Reply by NicoCaldo** [reply]
yeah, I would avoid to put an access token to a random page...

**Reply by KieranP** [reply]
@NicoCaldo The application is open source, so you're welcome to download the source code, inspect it, and run it locally if you are sceptical. Additionally, the instructions on the site encourage using a fine grained access token, so it only has access to repos you have starred, nothing else.

**Comment by MDshohel66** [comment]
Re: [community/community] [Feedback Wanted] Feed bug from 11/4/2024 - Feed
showing user activities from users not followed. (Discussion #143676)

**Comment by hundehausen** [comment]
Nothing has changed for me. It's still as bad as it can be. I haven't seen a single release in weeks now.

**Comment by KarbitsCode** [comment]
This is kind of concerning. Before the feed was just random showing releases, now it seems to be permanently not shown at all(?). My feed in the last week is literally always empty, despite setting the filter back and forth as a workaround.

**Reply by KieranP** [reply]
Until Github fixes this critically broken part of their app, for those who are growing weary of not being able to keep up with recent releases for starred repositories, I have created a service that does just that.

It is available here: **https://kieranp.github.io/Github-Releases-Feed/**

The service is open sourced under GPLv3 here: https://github.com/KieranP/Github-Releases-Feed

Hope this is useful to others. If you find it useful, feel free to star the repo :-)

**Reply by KarbitsCode** [reply]
> https://kieranp.github.io/Github-Releases-Feed/

Hey, this is actually good. Thanks for this 👍

**Reply by KarbitsCode** [reply]
There is another option that I currently use in here: https://github.com/orgs/community/discussions/145214#discussioncomment-11514154
It works well, but requires more work to set up and find the feed reader for desktop.
Just in case if anyone curious.

**Comment by OneSeven** [comment]
@LiteBrite82 

I starred thousands of repositories, but now my feed page does not show any dynamics. No matter how I set the filter, I try to switch the filter with any conditions every day, but I can't get the dynamics of the repository. This situation has seriously affected my daily development work. I am now disgusted with GitHub, especially when GitHub was acquired by Microsoft. This kind of low-level error should not appear! ! ! !

![1](https://github.com/user-attachments/assets/338f8658-b01d-40ec-b0b0-0151e9861d44)

**Reply by KieranP** [reply]
Until Github fixes this critically broken part of their app, for those who are growing weary of not being able to keep up with recent releases for starred repositories, I have created a service that does just that.

It is available here: **https://kieranp.github.io/Github-Releases-Feed/**

The service is open sourced under GPLv3 here: https://github.com/KieranP/Github-Releases-Feed

Hope this is useful to others. If you find it useful, feel free to star the repo :-)

**Comment by DenuxPlays** [comment]
Not working for me.
I starred ~100 repositories and there should be at least a few releases there.
The only thing I see is that a user I follow has created a repository.

**Reply by KieranP** [reply]
Until Github fixes this critically broken part of their app, for those who are growing weary of not being able to keep up with recent releases for starred repositories, I have created a service that does just that.

It is available here: **https://kieranp.github.io/Github-Releases-Feed/**

The service is open sourced under GPLv3 here: https://github.com/KieranP/Github-Releases-Feed

Hope this is useful to others. If you find it useful, feel free to star the repo :-)

**Comment by Yangruipis** [comment]
use https://github.com/dashboard-feed as a workaround

**Reply by OneSeven** [reply]
> use https://github.com/dashboard-feed as a workaround

It doesn't work for me

![3](https://github.com/user-attachments/assets/f7385f6b-7aa8-4f7c-b2f2-cd714707fc65)

**Reply by NicoCaldo** [reply]
I can confirm it doesn't work for me as well

**Reply by huyhoang160593** [reply]
Not working for me

**Reply by KieranP** [reply]
Until Github fixes this critically broken part of their app, for those who are growing weary of not being able to keep up with recent releases for starred repositories, I have created a service that does just that.

It is available here: **https://kieranp.github.io/Github-Releases-Feed/**

The service is open sourced under GPLv3 here: https://github.com/KieranP/Github-Releases-Feed

Hope this is useful to others. If you find it useful, feel free to star the repo :-)

**Reply by xyloflake** [reply]
@KieranP superb! Thanks for the awesome repo :)

**Comment by troysjanda** [comment]
my feed is oldest 1st and not the newest, I think this is tiresome to bo honest I want to see the newest items 1st not the oldest

**Reply by xyloflake** [reply]
Can confirm this indeed did happen for me and @gamersi

**Reply by KieranP** [reply]
Until Github fixes this critically broken part of their app, for those who are growing weary of not being able to keep up with recent releases for starred repositories, I have created a service that does just that.

It is available here: **https://kieranp.github.io/Github-Releases-Feed/**

The service is open sourced under GPLv3 here: https://github.com/KieranP/Github-Releases-Feed

Hope this is useful to others. If you find it useful, feel free to star the repo :-)

**Comment by Michael21107** [comment]
only works if the ''stars'' filter is unchecked
update: seems like it worked with default (all) filters but unchecking recommendations broke it

**Comment by zt64** [comment]
I'm still having issues with it not showing starred repo releases or announcements. No amount of changing the filters fixes it

**Reply by KieranP** [reply]
Until Github fixes this critically broken part of their app, for those who are growing weary of not being able to keep up with recent releases for starred repositories, I have created a service that does just that.

It is available here: **https://kieranp.github.io/Github-Releases-Feed/**

The service is open sourced under GPLv3 here: https://github.com/KieranP/Github-Releases-Feed

Hope this is useful to others. If you find it useful, feel free to star the repo :-)

**Comment by doggycxj** [comment]
still issues

**Comment by IvanLi-CN** [comment]
I look at the dashboard almost every day, but for the last month I've been barely able to refresh the items I've starred? For the last two weeks I've only seen the long overdue Release stream last weekend and then it didn't work again.

**Reply by KieranP** [reply]
Until Github fixes this critically broken part of their app, for those who are growing weary of not being able to keep up with recent releases for starred repositories, I have created a service that does just that.

It is available here: **https://kieranp.github.io/Github-Releases-Feed/**

The service is open sourced under GPLv3 here: https://github.com/KieranP/Github-Releases-Feed

Hope this is useful to others. If you find it useful, feel free to star the repo :-)

**Reply by IvanLi-CN** [reply]
This project saved me, and now I can finally see how much I've missed. Extremely grateful

**Comment by aspeckofdustin** [comment]
Hi there, reaching out from the Sunset product team here!  I'm newer to our Feed team so I'm learning about all the nuances, but we most certainly should not be showing you all activities from users that are not followed.  We've added this onto the backlog as a team and will be sorting through this as part of upcoming releases in the first half of 2025.

I also noticed that there were a few other callouts here, such as not being able to refresh the items starred and not seeing starred repos.  I'll be sharing this with our team that's working on the UI of the dashboard and see if we can prioritize a fix.  At the first sign of an update, I'll come back and share with you all!

Thanks for taking the time to keep it on our radar.

❤️ 
Dustin

**Reply by NicoCaldo** [reply]
News? The feed still won't work

**Comment by ADD-SP** [comment]
I'm still observing the `release` activities of some users I'm not following.

**Comment by imthatgin** [comment]
I keep getting my feed full of "starred" activity from someone I've previously sponsored but I do not follow. There seems to be no way to ignore that specific event from that user, without completely disabling stars.

**Reply by amol-** [reply]
I see commits and pull requests from someone I sponsored in the past and that I no longer sponsor. Couldn't find a way to stop them.

**Comment by bkbartk** [comment]
I get nightlies from wled. 
fir this repo I would just want to disable pre releases. But I can only disable releases from 1maintainer or instant the repo

**Comment by jstrieb** [comment]
I am also seeing items in my feed from those I do not follow. Unlike others in this thread: the activity I am seeing is almost exclusively from organizations (as opposed to individual users), and the activity is for repositories I have never interacted with in any way – not by starring the repo nor by following the parent user/organization.

![image](https://github.com/user-attachments/assets/7eac8129-dce6-4e4c-884e-1e2f55245a94)

What's interesting about the included image here is that I don't follow `golang` and have never starred any of its repositories, but I do follow `ianlancetaylor`. Perhaps the feed (in my case) is incorrectly reporting why the activity is showing up?

Update: I hit the "unfollow this user" button in the three dots menu within the feed, and it unfollowed `ianlancetaylor`, so probably it's just mislabeled.

**Comment by lemonyueya** [comment]
IT'S STILL HAPPENING!

**Comment by mootari** [comment]
I don't follow anyone, but I'm still seeing activity for a user that I sponsored once.

**Comment by milanglacier** [comment]
I am constantly receiving unwanted feed updates from a user, jose-elias-alvarez, whom I **do not follow, sponsor**, or **have starred repositories** from.

As the screenshot shows there's no reason explaining why I am receiving feeds from him.

 I request that GitHub's team resolve this issue promptly so that I no longer receive these updates from him.

I have no personal issue with Jose, I respect his contribution to the open source community. It is just that our development interests simply do not align.

<img width="770" alt="Screenshot 2025-03-25 at 12 06 45" src="https://github.com/user-attachments/assets/2bc6d282-ed1a-414c-988c-db2654ee0063" />

**Reply by heinermann** [reply]
I am having this problem as well with someone I once sponsored but no longer do (user: ds5678).

1. I don't follow them or sponsor them.
2. None of the repos involved are followed or starred.
3. Sponsors and recommendations activity filters are disabled.

**Reply by aspeckofdustin** [reply]
Hi @milanglacier -- thanks for sharing details to help us troubleshoot with you.  I was curious if it's similar to what @heinermann is experiencing, did you _ever_ Sponsor that user?  Or the handle is unrecognizable to you?

**Reply by milanglacier** [reply]
I once followed the user  jose-elias-alvarez. But I have unfollowed this user for over than two years.

**Comment by tamirzb** [comment]
Hi @aspeckofdustin you seem to handling this from the GitHub side. I am still facing the issue where my feed is full of people I sponsored in the past and I have no way of removing them from the feed. Are there any updates regarding fixing this issue?

**Comment by surmish** [comment]
I'm seeing updates from users sponsored in the past but can't disable their updates showing up on my feed. (E.g. my feed is showing updates from BurntSushi but I don't follow him anymore) Is there any other data point you need, @aspeckofdustin ?
